README.md

```diff
@@ -1,4 +1,10 @@
-# Consul [![CircleCI](https://circleci.com/gh/hashicorp/consul/tree/main.svg?style=svg)](https://circleci.com/gh/hashicorp/consul/tree/main) [![Discuss](https://img.shields.io/badge/discuss-consul-ca2171.svg?style=flat)](https://discuss.hashicorp.com/c/consul)
+<p>
+  <a href="https://consul.io" title="Consul website">
+    <img src="./website/public/img/logo-hashicorp.svg" alt="Consul logo" width="200px">
```

Comment:
I did it this way rather than the standard GH Markdown approach for images so that I could control the asset width. The logo doesn't need to fill the full width - that's excessive.